src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -59,6 +64,7 @@ private AccordService()
                              new AccordAgent(),
                              scheduler,
                              AccordCommandStores::new);
+        this.nodeShutdown = toShutdownable(node);
```

Comment:
don't like this, but will send a PR for accord to cleanup a few things (shutdown able, gradle changes, etc.)